ocs_samples/basic_samples/SDS/DotNet/SdsClientLibraries/SdsClientLibraries/Program.cs

```diff
@@ -452,7 +460,7 @@ public static async Task<bool> MainAsync(bool test = false)
 
 
 
```

Comment:
bunch of empty lines here also extra empty lines on 448, 437, 433, 429, 575 and 630